@types/index.d.ts

```diff
@@ -35,3 +36,47 @@ type LoginResponse = {
   };
   status: number;
 };
+
+type Venue = {
+  id: number;
+  capacity: number;
+  closing_time: string;
+  name: string;
+  opening_time: string;
+  venue_type: "Pub" | "Bar" | "Restaurant";
+  address: Address;
+  attributes: Attribute[];
+  beverages: Beverage[];
+};
+
+type VenueResponse = {
+  data: {
+    data: Venue[];
```

Comment:
how come this is nested like this?

Reply:
That is the shape of the data when it comes out of the api.